offices/_single.html

```diff
@@ -1,8 +1,11 @@
 {% extends "layout-side-nav.html" %}
 {% import "_vars-offices.html" as vars with context %}
 
+{% set office_title = office.title | safe if office.title %}
```

Comment:
I second this. I don't think it's necessary to assign these variables when the `office` already contains them so conveniently.